### DISCUSSION TITLE
Please enable 'MESSAGE on' on AIS-catcher settings.

### DISCUSSION THREAD

**Discussion by Taxom** [jvde-github/AIS-catcher#105] [Q&A] [answered]
How to enable the function of showing the last message?
Thank you.

upd: Sorry, I found it myself. It was just necessary to add "MESSAGE on" after  -N in config file.

**Comment by jvde-github** [comment, marked as answer]
Apologies, still undocumented as it is new. I have improved the error message adding -N and also allow now to use `-N MSG on` and `-N MESSAGE on` to keep the command line slightly more compact.
